### DISCUSSION TITLE
React Native New Architecture support

### DISCUSSION THREAD

**Discussion by jorgemartiinez** [revtel/react-native-nfc-manager#751] [General]
Hi,
Sorry if this is not the place to ask, but I haven't found any other place to do so. Does, or will this library support React Native New Architecture? Because I'm just starting a big project that will need to use this library and I have to know this to either use the new architecture right now or wait to do that.
Thanks and sorry for inconvenience,

**Comment by dcanora** [comment]
[This comment](https://github.com/revtel/react-native-nfc-manager/issues/766#issuecomment-2605330629) on an issue seems to indicate it does support the new architecture.
